apm-sniffer/apm-sdk-plugin/lettuce-5.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/lettuce/v5/RedisChannelWriterInterceptor.java

```diff
@@ -61,6 +71,20 @@ public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allAr
         Tags.DB_STATEMENT.set(span, dbStatement.toString());
         SpanLayer.asCache(span);
     }
+    
+    private String getArgsStatement(RedisCommand redisCommand) {
```

Comment:
Take a look at `JDBCPluginConfig#TRACE_SQL_PARAMETERS` to do. Collecting parameters costs a lot of resources (CPU/memory).

Reply:
And you need to update the tests to verify the parameter collection here. https://github.com/apache/skywalking/tree/master/test/plugin/scenarios/redisson-scenario